doc/guide/startup.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
+<!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.3//EN"
+	"http://www.oasis-open.org/docbook/xml/4.3/docbookx.dtd">
+<chapter id="startup">
+  <title>Start up</title>
```

Comment:
should we explain this a bit further?
e.g. `Start up (cockpit services, processes)`